sdk/src/wallet/storage/constants.rs

```diff
@@ -4,16 +4,16 @@
 /// The default storage path.
 pub const DEFAULT_STORAGE_PATH: &str = "./storage";
 
-/// The default RocksDB storage path.
-#[cfg(feature = "jammdb")]
+/// The default jammDB storage path.
+#[cfg(all(not(feature = "rocksdb"), feature = "jammdb"))]
 pub(crate) const JAMMDB_FOLDERNAME: &str = "walletdb";
 
 /// The default RocksDB storage path.
 #[cfg(feature = "rocksdb")]
 pub(crate) const ROCKSDB_FOLDERNAME: &str = "walletdb";
 
 pub const fn default_storage_path() -> &'static str {
-    #[cfg(all(not(feature = "jammdb"), feature = "rocksdb"))]
+    #[cfg(feature = "rocksdb")]
```

Comment:
This could benefit from `cfg_if` too

Reply:
Unable to understand your comment and what exactly is expected here.

Reply:
I have pushed some modifications. You can review them now.